src/generation.rs

```diff
@@ -1451,20 +1481,39 @@ impl GenerationScope {
                 if CLI_ARGS.preserve_encodings {
                     deser_code.content
                         .line(&format!("let len = {}.array_sz()?;", deserializer_name))
```

Comment:
Ignore this added `mut`. If we need to make any other changes to this PR I''ll remove it in that commit, otherwise we can remove it when we fix the other warnings in `cddl-codegen`.